docs/Using Price Feeds/feed-registry.html

```diff
@@ -55,7 +52,7 @@
 
 ## Denominations library
 
-A <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/Denominations.sol" target="_blank">`Denominations`</a> Solidity library is available for you to query common denominations:
+A <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/Denominations.sol" target="_blank">`Denominations`</a> Solidity library is available for you to query currency identifiers which lack a canonical Ethereum address:
 
 
 ```javascript Solidity 8
```

Comment:
Replace javascript with solidity for all code samples. We added solidity code styling a while ago.